src/main/java/org/isf/patient/gui/PatientSummary.java

```diff
@@ -160,10 +163,18 @@ private JPanel getPatientCard() {
 		JLabel patientAge = new JLabel(MessageBundle.getMessage("angal.common.age.txt") + ": " + TimeTools.getFormattedAge(patient.getBirthDate()));
 		JLabel patientSex = new JLabel(MessageBundle.getMessage("angal.common.sex.txt") + ": " + patient.getSex());
 		JLabel patientTOB = new JLabel(MessageBundle.getMessage("angal.patient.tobm") + ": " + filtra(patient.getBloodType()));
-		
+
 		JLabel patientPhoto = new JLabel();
-		if (patient.getPatientProfilePhoto() != null && patient.getPatientProfilePhoto().getPhotoAsImage() != null) {
-			patientPhoto.setIcon(new ImageIcon(ImageUtil.scaleImage(patient.getPatientProfilePhoto().getPhotoAsImage(), GeneralData.IMAGE_THUMBNAIL_MAX_WIDTH)));
+		PatientProfilePhoto photo = null;
+		try {
+			photo = this.patientBrowserManager.retrievePatientProfilePhoto(patient);
+		} catch (OHServiceException e) {
+			OHServiceExceptionUtil.showMessages(e);
+		}
+
+		if (photo != null && photo.getPhotoAsImage() != null) {
+			patientPhoto.setIcon(
+							new ImageIcon(ImageUtil.scaleImage(photo.getPhotoAsImage(), GeneralData.IMAGE_THUMBNAIL_MAX_WIDTH)));
```

Comment:
surprised this is on a new line

Reply:
it is the formatter doing this, if you fix and save the same file, what is it doing in your IDE?